javascript/stimulus_reflex.js

```diff
@@ -39,34 +45,26 @@ class StimulusReflexController extends Controller {
 //
 const initialize = (
   application,
-  { controller, consumer, debug, params, isolate, deprecate } = {}
+  { controller, consumer, debug, params, isolate, deprecate, transport } = {}
 ) => {
-  ActionCableTransport.initialize(consumer, params)
-  document.addEventListener(
-    'DOMContentLoaded',
-    () => {
-      if (Deprecate.enabled && IsolationMode.disabled)
-        console.warn(
-          'Deprecation warning: the next version of StimulusReflex will standardize isolation mode, and the isolate option will be removed.\nPlease update your applications to assume that every tab will be isolated.'
-        )
-    },
-    { once: true }
-  )
+  Transport.set(transport || ActionCableTransport)
+  Transport.plugin.initialize(consumer, params)
```

Comment:
Great to see this. Excited to explore how we can capitalize on different transports.